package.json

```diff
@@ -10,7 +10,7 @@
     "build-lib": "npm run build-form && babel src -d lib --source-maps --extensions '.ts,.js'",
     "build-dev": "webpack --progress --mode=development",
     "build-types": "tsc --emitDeclarationOnly",
-    "build-version": "./timestamp.sh > src/versionInfo.ts  && eslint 'src/versionInfo.ts' --fix",
```

Comment:
This replaces whatever the build-version script did before by calling timestamp.sh with just a call to eslint. Does not seem right to me.

Reply:
Not sure as to why VSCode throws off that error, but in the mean time, perhaps we could use it as a cautionary suggestion to Windows 10 users? If it doesn't seem right, then maybe it is not. Should the suggestion instead go the readme?

Reply:
Currently the dev setup seems not to be Windows compatible. At least the timestamp.sh would need to change if we want to make the build run under windows. There might be more changes needed. Under Windows you could try using WSL.